### DISCUSSION TITLE
Are themes that includes JS allowed?

### DISCUSSION THREAD

**Discussion by SoftwareType** [FirefoxCSS-Store/FirefoxCSS-Store.github.io#284] [Q&A] [answered]
Themes that includes chrome folder are always mentioned as a FirefoxCSS instead of anything else, due to how popular it became. However, rule 3 exists in the subreddit to be faithful to its name

Will this also apply on this github page?

**Comment by Neikon** [comment, marked as answer]
I'm not sure if I understand the question correctly, are themes with js allowed in this repo? Yes, as long as their code is public, we don't pretend nor have the ability to review themes or guarantee security, that's why we don't host any themes ourselves and just link to the original repo.

**Reply by BeyondMagic** [reply]
Note that due to the Reddit's rule, we not following it may have our website removed from their list, although I doubt it. It's not a rule that should exist at all, anyway. JS allows for greater customization and personalization.

**Reply by SoftwareType** [reply]
> Note that due to the Reddit's rule, we not following it may have our website removed from their list, although I doubt it. It's not a rule that should exist at all, anyway. JS allows for greater customization and personalization.

I suppose that's because of past JavaScript hacks, but I'm still not sure about it
UPDATE: I suppose you can just check for anything by using antivirus
